spec/models/plant_accession_spec.rb

```diff
@@ -47,6 +47,25 @@
     end
   end
 
+  context 'unique attributes' do
+    it 'disallows duplicate plant_accession and originating_organisation' do
```

Comment:
Yeah, it is not obvious how to do that, but it's described in the docs I linked before:

```ruby
RSpec.describe Post, type: :model do
  describe "validations" do
    subject { FactoryGirl.build(:post) }
    it { should validate_uniqueness_of(:title) }
  end
end
```